actors/evm/src/interpreter/system.rs

```diff
@@ -205,40 +191,28 @@ impl<'r, RT: Runtime> System<'r, RT> {
     }
 
     /// Get value of a storage key.
-    pub fn get_storage(&mut self, key: U256) -> Result<Option<U256>, StatusCode> {
-        let mut key_bytes = [0u8; 32];
-        key.to_big_endian(&mut key_bytes);
-
-        Ok(self.slots.get(&key).map_err(|e| StatusCode::InternalError(e.to_string()))?.cloned())
+    pub fn get_storage(&mut self, key: U256) -> Result<U256, StatusCode> {
+        Ok(self
+            .slots
+            .get(&key)
+            .map_err(|e| StatusCode::InternalError(e.to_string()))?
```

Comment:
get of something not existing is just zero.

Reply:
Yes. In the EVM, all storage "exists", it's just all zeros.

Reply:
Same re test.

Reply:
See the fixed test below.